packages/ui/Layout/Cell.js

```diff
@@ -96,6 +96,8 @@ const CellBase = kind({
 		 */
 		grow: PropTypes.bool,
 
+		key: PropTypes.string,
```

Comment:
here I need help to properly describe this prop

Reply:
My suggestion is:
`
/**
	* A unique key used by React to identify the component.
        * @type {String}
        * @public
	 */
`
but let's see SWP team's opinion.

Reply:
I think we can remove this docs.